doc/decisions/decision_process.md

```diff
@@ -70,6 +72,8 @@ We base our decision process and template on:
 Following subsections describe all steps a decision might run through.
 Only two of them are mandatory.
 
+In all steps we avoid discussions in issues or PRs but directly update the decision text with the different opinions and discuss the text of the decision PR.
+
```

Comment:
I rephrased the sentence 77a90b01bc2aee827391520c6a2c5824f06ff1e5 and removed a useless restriction in the notes.

Reply:
tl;dr

But whatever the problem is, there is a simple solution:
I added my constraints in 1c8d75fd5f665d3ba4f56e5c0827716c44db6d0b

Please make suggestions of what your constraints are so that you can participate in decisions (what I definitely want). Then we adapt the decision process to the constraints we personally have.